docs/sources/access/index.md

```diff
@@ -0,0 +1,87 @@
+---
+description: Access or install Explore Traces.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/tempo/access/
+keywords:
+  - Install
+  - Configure
+  - Explore Traces
+menuTitle: Access or install
+weight: 150
+refs:
+  tempo-data-source:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/datasources/tempo/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/
+---
+
+# Access or install Explore Traces
+
+{{< docs/public-preview product="Explore Traces" >}}
+
+You can access Explore Traces using Grafana Cloud or a self-managed OSS Grafana or Grafana Enterprise.
+
+The easiest way to access Explore Traces is in Grafana Cloud. No setup or installation is required.
+
+To use Explore Traces with self-managed Grafana, you need to install the Explore Traces plugin.
+
+Explore Traces requires Tempo 2.6 or newer.
+
+## Access in Grafana Cloud
+
+To use Explore Traces with Grafana Cloud, you need:
+
+- A Grafana Cloud account
+- A Grafana stack in Grafana Cloud with a configured [Tempo data source](https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/tempo/configure-tempo-data-source/) receiving tracing data
+
+To access Explore Traces:
+
+1. Open your Grafana stack in a web browser.
+1. In the main menu, select **Explore** > **Traces**.
+
+## Access in self-managed Grafana
+
+To use Explore Traces with self-managed Grafana open source or Grafana Enterprise, you need:
+
+- Your own Grafana instance running 11.2 or newer
+- Tempo 2.6 or newer
+- A configured [Tempo data source](https://grafana.com/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/) receiving tracing data
+
+Once installed, you can access Explore Traces:
```

Comment:
I've moved the Access steps to the bottom and have reworked the other sections to be set up instead of access. I think that makes it clearer. thank you for the suggestions!